aws/resource_aws_guardduty_member.go

```diff
@@ -93,11 +176,23 @@ func resourceAwsGuardDutyMemberRead(d *schema.ResourceData, meta interface{}) er
 		d.SetId("")
 		return nil
 	}
+
 	member := gmo.Members[0]
 	d.Set("account_id", member.AccountId)
 	d.Set("detector_id", detectorID)
 	d.Set("email", member.Email)
 
+	status := aws.StringValue(member.RelationshipStatus)
+	d.Set("relationship_status", status)
+
+	//https://docs.aws.amazon.com/guardduty/latest/ug/list-members.html
+	if status == "Created" || status == "Resigned" {
+		d.Set("invited", false)
+	} else if status == "Disabled" || status == "Enabled" ||
```

Comment:
The attribute for setting here is `invite` not `invited` and we should check for `status == "Invited"`. I'm not sure if we should consider either of the email verification statuses as "sent" since it can fail (which means it should probably show as needing to re-invite the member). I believe this logic would probably be simpler as:

```go
d.Set("invite", false)
if status == "Disabled" || status == "Enabled" || status == "Invited" {
  d.Set("invite", true)
}
```

Reply:
Hmm, how about `EmailVerificationInProgress`? I believe it means awaiting status for member's response. Anyway, I think I should include which `relationship_status` from AWS is counted as `invited` status in Terraform.